### ISSUE TITLE
clearInstance bug

### ISSUE THREAD

**Issue by shingohu** [infitio/flutter_socket_io#13]
```
///Disconnect a socket instance and remove from stored sockets list
  Future clearInstance(SocketIO socket) async {
    await _channel.invokeMethod('clearInstance', {'id': socket.id});
    _sockets.remove(socket.id);
  }
```
```
case "clearInstance":
                if(arguments["index"] == nil){
                    result(FlutterError(code: "400", message: "Invalid instance identifier provided", details: nil))
                }
```
the "index" should be "id"

**Comment by tiholic** [comment]
@shingohu thanks for pointing out!

**Comment by tiholic** [comment]
fixed in https://github.com/infitio/flutter_socket_io/commit/a7b60f07f8e7fb0ff8e177730ebe1d31553b779f

Published to pub: [`0.1.10`](https://pub.dartlang.org/packages/adhara_socket_io/versions/0.1.10)

**Comment by liuyun1014** [comment]
clearInstance don't work for ios
adhara_socket_io: ^0.2.0
